docker-compose.yaml

```diff
@@ -7,14 +7,21 @@ services:
       - 8080:8080
       - 5001:5001
   vitalam-node:
-    image: ghcr.io/digicatapult/vitalam-node:v2.4.0
+    image: ghcr.io/digicatapult/vitalam-node:v2.5.0
+    command: /bin/sh -c "
+      ./vitalam-node 
+      --base-path /data/ 
+      $$IDENTITY
+      $$WS
+      $$RPC
+      $$CORS"
     ports:
       - 30333:30333
       - 9944:9944
       - 9933:9933
     environment:
-      - IDENTITY=dev
-      - WS=true
-      - RPC=true
-      - CORS=all
+      - IDENTITY=--dev
+      - WS=--unsafe-ws-external
+      - RPC=--unsafe-rpc-external
+      - CORS=--rpc-cors all
```

Comment:
Do you not want to pass these values as args directly in the command above?

Reply:
Agreed not sure why we're using envs here. It'll be simpler to just set them directly

Reply:
changed to use directly